tests/ui_tests/app/configuration.py

```diff
@@ -324,3 +329,7 @@ def check_that_fields_and_group_are_invisible(self):
             assert not field.is_displayed(), field.get_attribute("class")
         group_names = self.get_group_elements()
         assert not group_names, group_names
+
+    @allure.step('Refresh configuration')
+    def refresh(self):
+        self.__init__(self.driver, self.url)
```

Comment:
Why not use `driver.refresh()`?

Reply:
@vgorkavenko because of `__init__` of the `Configuration` class

Reply:
`__init__` routine extracted to the separate method and used alongside suggested `driver.refresh()`